source/_integrations/motion_blinds.markdown

```diff
@@ -27,6 +27,7 @@ Additionally the following brands have been reported to also work with this inte
 - [Bloc Blinds](https://www.blocblinds.com/)
 - [Brel Home](https://www.brel-home.nl/)
 - [3 Day Blinds](https://www.3dayblinds.com/)
+- [Decorquip Dream](https://www.decorquip.com/post.php?dream)
```

Comment:
The others are all virtual integrations, why isn't this one added as such?

../Frenck